addons/sourcemod/scripting/vsh/abilities/ability_rage_gas.sp

```diff
@@ -0,0 +1,191 @@
+#define BOMBPROJECTILE_MODEL	"models/props_lakeside_event/bomb_temp.mdl"
```

Comment:
I'd move this define to `saxtonhale.sp` as it also defined in `ability_rage_bomb_projectile.sp`